runtime/core/error.h

```diff
@@ -93,6 +93,9 @@ enum class Error : error_code_t {
   /// Execute stage: The handle is invalid.
   DelegateInvalidHandle = 0x32,
 
+  /// Backend registration: trying to access an invalid index in the backend
+  /// registration table.
+  BackendTableAccessFail = 0x33,
```

Comment:
I wouldn't add a new error message, just for this. InvalidArgument = 0x12 is sufficient for this case.

Reply:
Sure